app/views/admin/tracks/index.html.haml

```diff
@@ -6,59 +6,61 @@
         Categorize events in your conference
 .row
   .col-md-12
-    %table.table.table-hover#tracks
+    %table.table.table-hover.table-striped.table-bordered.datatable#tracks
       %thead
         %th Name
```

Comment:
We should add a field like ID, in order to be able to see which one is the oldest request, and which one is the newest.